third_party/move/move-model/bytecode/src/demove_peephole_analysis.rs

```diff
@@ -0,0 +1,511 @@
+// Copyright (c) Verichains
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+use move_model::model::FunctionEnv;
```

Comment:
If its `demove` it should go the decompiler tree.

Reply:
fixed